Errors.cpp

```diff
@@ -0,0 +1,17 @@
+#include "Errors.hpp"
+
+ToLowSpeedError ::
+    ToLowSpeedError(std::string speed)
+    : std::logic_error("To Low Speed Error - speed " + speed + " cannot be set for this car.") {}
+
+ToHighSpeedError ::ToHighSpeedError(std::string speed)
+    : std::logic_error("To High Speed Error - speed " + speed + " cannot be set for this car.") {}
+
+AccelerateToLoweSpeedError ::AccelerateToLoweSpeedError(std::string speed)
+    : std::logic_error("You cannot accelerato to lower speed than actual (" + speed + ").") {}
+
+BrakeToHigherSpeedError ::BrakeToHigherSpeedError(std::string speed)
```

Comment:
strange and inconsisten formatting of constructor scopes. Use `A::A`, it is the most common way.